src/utils/dispute-utils.js

```diff
@@ -372,7 +374,10 @@ function getRoundPhasesAndTime(courtConfig, round, currentPhase) {
     },
     {
```

Comment:
Might be useful to add a few comments ahead of both the `AppealRuling` and `ConfirmAppeal` objects to describe how they should be changing based on interactions / on-chain conditions.

It took me a minute to realize that using `appeal.createdAt` in `AppealRuling` made sense (as this is always a date in the past), because if an appeal was created, it means we should be past this stage already.